src/releaseAndTag.js

```diff
@@ -107,7 +106,7 @@ const createTempDirectory = async (name) => {
   if (exists) {
     throw new Error(`Tried to create directory, but it already exists: ${dir}`);
   }
-  mkdirp(dir);
+  fs.mkdirSync(dir, { recurseive: true });
```

Comment:
I don't think this code is exercised by tests, so some manual checking that I didn't mess everything up would be a good thing.

Reply:
Should probably be this?

```suggestion
  await fs.mkdir(dir, { recursive: true });
```